app.js

```diff
@@ -88,27 +89,54 @@ app.post('/hangouts', (request, response) => {
 	}).catch(error(response));
 });
 
-app.post('/reportEvent', (request, response) => {
+app.post('/reportEvent', async (request, response) => {
+	const fhirUrl = request.body.fhirUrl;
+	if (!fhirUrl) {
+		response.status(400).send("missing fhirUrl");
+		return;
+	}
+	if (!settings.authorizedFhirUrls.includes(fhirUrl)) {
+		response.status(401).send(`unauthorized fhirUrl ${fhirUrl}`);
+		return;
+	}
+
+	const fhirAccessToken = request.body.fhirAccessToken;
```

Comment:
why not use Authorization header to pass the fhirAccessToken?

Reply:
The access token should be paired with a fhir base URL. I think it's better to put them at the same place, i.e., HTTP body.

Reply:
I see. It doesn't feel like a common practice though. Anyway, it is fine. I was just curious.